### PR TITLE
Added bower.json

### LINKED ISSUE #3: Add a bower.json
Would make it easier to include.

### PR DESCRIPTION
**Pull request by motorcityadam**
Added a bower.json to allow me to use grunt-bower-task to include this project as a dependency on one of my projects. I had to increase the version to 0.5.2 to repackage the release for Bower. This Closes #3.

### CODE REVIEW

**Comment by oste** [comment]
+1 for a merge

File: detect-element-resize.js [hunk 1]
```diff
@@ -35,6 +36,13 @@
 			expandChild.style.height = expand.offsetHeight + 1 + 'px';
 			expand.scrollLeft = expand.scrollWidth;
 			expand.scrollTop = expand.scrollHeight;
+			if( contract.scrollTop == 0 || contract.scrollLeft == 0 )
+			{
```
Comment (djmccormick) [inline detect-element-resize.js:40]: Doesn't match the style in the rest of the project.

**Comment by djmccormick** [comment]
+1

**Comment by sdecima** [comment]
Thanks for the contribution, I'm reviewing all the changes now.

**Comment by mtraynham** [comment]
+1 on the `display:none` detection, I thought it was related to that, but glad to see someone else has confirmed.

Might I suggest a different solution, a trick I learned from jQuery and element dimensions.  When you ask for the `offsetWidth` on an element and it is `display:none`, you'll get 0.  When you ask jQuery for the `$('#...').width()` on a `display:none` object, it temp sets the display to `block`, grabs the width and sets it back to `none`.  This would all happen between the redraw interval, so the user won't even notice.  

Maybe you could do something similar instead of polling.

**Comment by sdecima** [comment]
I finally merged most of the changes proposed in this pull request, and changed the polling with a CSS animation to detect when the element is visible as discussed with @thomassuckow.

**Pull request closed without merging**